registry/storage/driver/gdrive/gdrive_test.go

```diff
@@ -0,0 +1,135 @@
+package gdrive
+
+import (
+	"os"
+	"testing"
+
+	ctx "github.com/docker/distribution/context"
+	storagedriver "github.com/docker/distribution/registry/storage/driver"
+	"github.com/docker/distribution/registry/storage/driver/testsuites"
+	check "gopkg.in/check.v1"
+)
+
+func Test(t *testing.T) { check.TestingT(t) }
+
+var gdriveDriverConstructor func(rootDirectory string) (storagedriver.StorageDriver, error)
+var skipGdrive func() string
+
+func init() {
+
+	keyFile := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS_FILE")
```

Comment:
It would be advantageous to be able to have the contents of the file be in an environment variable and placed into a tempfile for the tests. The reasoning is that it will be easier for running in a CI environment if it does not need to rely on a file existing.